Enterprise-CMCS/macpro-mdct-mfp
# State & Territory-Specific Initiatives UI

### Description

Frontend setup for State & Territory-Specific Initiatives, contains Add Initiatives UI from @ajaitasaini & some temporary buttons for @cassandradanger.

Still needs routing and data put/get.

### Related ticket(s)


MDCT-2775
MDCT-2795
MDCT-2813

---
### How to test

Go to path _/wp/state-and-territory-specific-initiatives/initiatives_  
click around to confirm the UIs looks like the mockups:
- Add initiative
- Delete
- Edit name
- Edit (goes to a shell page)

### Important updates


N/A

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
